packages/strapi-parts/src/Accordion/Accordion.stories.mdx

```diff
@@ -83,7 +94,7 @@ Description...
 
 ## Accordion Group
 
-Description...
+The accordion component has two main states: collapsed and expanded. By default it has a collapsed state. If Call-to-actions are required, use them on the right side and  use chevron icon on the left one. 
```

Comment:
The accordion group story aims to show that we can wrap a set of accordion with a KeyboardNavigable component to make them keyboard navigable :)

Reply:
Is that something you'd like me to add, or a correction?